### ISSUE TITLE
Prefix verifier errors with ;

### ISSUE THREAD

**Issue by bjorn3** [CraneStation/cranelift#536]
This would allow copy pasting the verifier error for reducing.

```
[...]
    v12 = isub.i8 v11, v10
    store.i64 v9, v12
;   ^~~~~~~ verifier inst17: arg 1 (v12) with type i8 failed to satisfy type set ValueTypeSet { lanes: BitSet(1), ints: BitSet(96), floats: BitSet(0), bools: BitSet(0) }

    jump ebb9

ebb3:
[...]
```

**Comment by JoeHowarth** [comment]
Hi! Could someone point me where to look on this as a first issue?

**Comment by sunfishcode** [comment]
Hello! Take a look in lib/codegen/src/print_errors.rs, in `pretty_instruction_error` and `pretty_preamble_error`, where it prints the "^" arrows. A good way to start would be to factor out the arrow drawing code from both of those into a common helper function, and then teach the helper function to prepend the ";" (and adjust the indent level accordingly).

**Comment by JoeHowarth** [comment]
Thanks for the pointer

I tried `cargo run test path/to/filetest` hoping to see the ^~~~~ , but only got 
`FAIL ./filetests/verifier/globals.clif: 18: expected SSA value address` (I took out the ; that silenced the error)

What should I run to get the the above output instead?

**Comment by sunfishcode** [comment]
If you run `clif-util compile filetests/verifier/globals.clif`, you'll see the full verifier error.

**Comment by sunfishcode** [comment]
This is now fixed in #585!
